src/helix-ui/styles/components/typography.less

```diff
@@ -1,3 +1,64 @@
+/* ========== HEADINGS ========== */
```

Comment:
Added content came from `modules/typography.less` (git shows a move on `hljs.less` not `typography.less`)